egs/wsj/s5/local/nnet3/run_splstm.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# this is a state preserving lstm script
+# LSTM script runs for more epochs than the TDNN script
```

Comment:
delete these two lines of comments. Just have comments that are relevant to the state preserving LSTM.